canonical-sphinx-extensions/terminal-output/__init__.py

```diff
@@ -34,14 +35,12 @@ def input_line(prompt_text, command):
 
     def run(self):
 
-        # if :user: or :host: are provided, replace those in the prompt
+        # Build prompt with :user:, :host: and :dir: (whichever is provided)
 
-        command = "" if "input" not in self.options else self.options["input"]
```

Comment:
We still need the command (it's used in line 52).

But line 52 should be updated to only be included if command is not empty.